app/views/catalog/_document_list.html.erb

```diff
@@ -0,0 +1,12 @@
+<% # container for all documents in index list view -%>
+<div id="documents" class="documents-<%= document_index_view_type %>">
+  <% documents.in_groups_of(4, false) do |document_group| %>
+    <div class="row">
+      <% document_group.each do |document| %>
+        <div class="col-md-3">
+          <%= render document %>
+        </div>
+      <% end %>
+    </div>
+  <% end %>
+</div>
```

Comment:
@mccalluc I refactored this. See latest commit. May want to confirm visually before pushing. I don't have an index, and thus no search results.